lib/rubocop/ast/node/keyword_begin_node.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module AST
+    # A node extension for `kwbegin` nodes. This will be used in place of a plain
+    # node when the builder constructs the AST, making its methods available
+    # to all `kwbegin` nodes within RuboCop.
+    class KeywordBeginNode < Node
+      # Returns the body of the `kwbegin` block. Returns `self` if the `kwbegin` contains
+      # multiple nodes.
+      #
+      # @return [Node, nil] The body of the `kwbegin`.
+      def body
+        return unless node_parts.any?
+
+        if rescue_node
+          rescue_node.body
+        elsif ensure_node
+          ensure_node.node_parts[0]
```

Comment:
I'd like to do this more cleanly, but it's dependent on #334.

Reply:
Understood. I'd like to wait for the upcoming major release, though, so I'd say let's merge this now and tweak as part of #334